app/src/main/java/com/asfoundation/wallet/util/SyncExecutor.kt

```diff
@@ -0,0 +1,10 @@
+package com.asfoundation.wallet.util
+
+import java.util.concurrent.ScheduledThreadPoolExecutor
+
+
+class SyncExecutor : ScheduledThreadPoolExecutor {
+  constructor(corePoolSize: Int) : super(corePoolSize) {
+    this.maximumPoolSize = corePoolSize
+  }
+}
```

Comment:
Android Studio provide a nice feedback on this one.

This class can be simplified into:

class SyncExecutor(corePoolSize: Int) : ScheduledThreadPoolExecutor(corePoolSize)